pkg/gateway/operations/listobjects.go

```diff
@@ -21,8 +21,9 @@ import (
 
 const (
 	ListObjectMaxKeys = 1000
-	maxUploadsRange   = 2147483647
-	minUploadsRange   = 0
+	maxUploadsListMPU = 2147483647
+	minUploadsListMPU = 0
```

Comment:
1. if we leave the verification to the underlaying library - you can remove maxUploadsListMPU - it is MaxInt32 value and it will not pass this value as the cast will not let you.

2. if we plan to pass 0 as a valid value and let the underlying package handle it - we just need to check if maxUploads32 < 0